lms/djangoapps/philu_overrides/branding/api.py

```diff
@@ -32,7 +32,7 @@ def is_enabled():
     return BrandingApiConfig.current().enabled
 
 
-def get_auth_footer(is_secure=True):
+def get_auth_footer():
```

Comment:
Since we have removed the argument, lets also create a PR for theme side, to remove the argument from caller end as well.

Reply:
Surely, Already on it